en/docs/learn/querying-saml-assertions.md

```diff
@@ -32,29 +32,23 @@ authority.
 !!! note "Before you begin"
 
     1.  Clone or download the client application from [this GitHub
-        location](https://github.com/gayangithub/wso2-is-assertion-query-client)
-        and extract it. This folder location will be referred to as
+        location](https://github.com/wso2/samples-is),
+        extract it and navigate to
+        `           sample-saml-query-profile/saml-query-profile-client          `.
+        This folder location will be referred to as
         `            <CLIENT_HOME>           ` in this document.
 
     2.  Build the client application by executing the following command on
-        the `             <CLIENT_HOME>            ` root folder.
+        the `             <CLIENT_HOME>            ` folder.
 
         ``` java
         mvn clean install
         ```
 
 1.  Start the WSO2 Identity Server and log in to the management console.
-2.  Navigate to **Keystores** under **Manage** on the **Main** tab of
-    the management console and click **List**.
-3.  Click **Import Cert** to import a certificate to the
-    **wso2carbon.jks** keystore.  
-    ![import-cert-keystore](../assets/img/tutorials/import-cert-keystore.png)
-4.  Upload the certificate found in
-    `          <CLIENT_HOME>/src/test/resources/soa.cert         ` and
-    click **Import**.
-5.  Click **Add** under **Service Providers** on the **Main** tab.
-6.  Configure a service provider for SAML and click **Update**.  
-    To try querying an assertion using the a sample application,
+2.  Click **Add** under **Service Providers** on the **Main** tab.
+3.  Configure a service provider for SAML and click **Update**.
+    To try querying an assertion using the sample application,
     configure the service provider for the travelocity sample
     application and follow the steps in the
     [try out scenario](#try-out-scenario) section.
```

Comment:
To create a service provider that can be used to query an assertion, see [Try out scenario](#try-out-scenario).